client/my-sites/media/single.jsx

```diff
@@ -0,0 +1,119 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import { getSelectedSite } from 'state/ui/selectors';
+import DocumentHead from 'components/data/document-head';
+import PageViewTracker from 'lib/analytics/page-view-tracker';
+import { EditorMediaModalDetail } from 'post-editor/media-modal/detail';
+
+class SingleMediaComponent extends Component {
+	static propTypes = {
+		selectedSite: PropTypes.object,
+	};
+
+	state = {
+		currentDetail: null,
+		editedImageItem: null,
+		editedVideoItem: null,
+		selectedItems: [],
+		source: '',
+	};
+
+	// componentDidMount() {
+	// }
+	//
+	// componentDidUpdate( prevProps ) {
+	// }
+
+	render() {
+		const selectedItems = [
```

Comment:
Seems like we'll conveniently be able to use `<QueryMedia mediaId={ this.props.attachment } />` to fetch this information and the `getMediaItem` selector to retrieve it from the store.